core/src/stmt.rs

```diff
@@ -24,3 +24,55 @@ pub trait MongoStatement: Debug {
             .map_or(Err(Error::ColIndexOutOfBounds(col_index)), Ok)
     }
 }
+
+#[derive(Debug)]
+pub struct EmptyStatement {
+    pub resultset_metadata: Vec<MongoColMetadata>,
```

Comment:
```suggestion
    pub resultset_metadata: &'static Vec<MongoColMetadata>,
```
If you make this a static ref then there is no need for cloning the metadata and you can do this for example :
```
impl MongoPrimaryKeys {
    pub fn empty() -> EmptyStatement {
        EmptyStatement {
            resultset_metadata: &PK_METADATA,
        }
    }
}
```

Reply:
Oh, good idea.